<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLineEdit" name="SunEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="SunLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>修改阳光</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="SunBtn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QLabel" name="ColdLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>卡槽冷却</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="UserMessageWidget" native="true">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>0</y>
     <width>260</width>
     <height>511</height>
    </rect>
   </property>
   <widget class="QLabel" name="GameName">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>130</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>植物大战僵尸Steam版</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="C1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>卡槽1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="C2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>卡槽2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="C3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>卡槽3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="C4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>卡槽4</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="C5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>卡槽5</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="C6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>卡槽6</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="C7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>卡槽7</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="C8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>卡槽8</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="C9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>卡槽9</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="C10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>卡槽10</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="C1Edit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="C2Edit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="C3Edit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="C4Edit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="C5Edit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="C6Edit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>280</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="C7Edit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>320</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="C8Edit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>360</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="C9Edit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>400</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="C10Edit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>440</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="CardSelect">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ColdBtn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="PU">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>植物无敌</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="PS">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>植物秒杀</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="PlantBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>271</width>
     <height>90</height>
    </rect>
   </property>
   <property name="title">
    <string>种植植物</string>
   </property>
   <widget class="QLabel" name="PXlabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>植物X坐标</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="PYlabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>植物Y坐标</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="PX">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="PY">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="PIDLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>50</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>植物ID</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="PID">
    <property name="geometry">
     <rect>
      <x>168</x>
      <y>20</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="PSbtn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>90</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>确认放置</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="ZombieBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>271</width>
     <height>90</height>
    </rect>
   </property>
   <property name="title">
    <string>放置僵尸</string>
   </property>
   <widget class="QLabel" name="ZXlabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>僵尸X坐标</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ZYlabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>植物Y坐标</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="ZX">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="ZY">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ZIDLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>50</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>植物ID</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="ZID">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ZSbtn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>90</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>确认放置</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="OHBTN">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>450</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>狂热模式</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
